src/coreclr/tools/aot/ILCompiler/ILCompilerRootCommand.cs

```diff
@@ -315,89 +155,18 @@ <<<<<<< HEAD
             new("--singlemethodgenericarg") { Description = "Single method compilation: generic arguments to the method" };
         public CliOption<string> MakeReproPath { get; } =
             new("--make-repro-path") { Description = "Path where to place a repro package" };
+        public CliOption<string[]> WasmImport { get; } =
+            new("--wasmimport") { Description = "WebAssembly import module names for PInvoke functions"};
+        public CliOption<string[]> WasmImportList { get; } =
+            new("--wasmimportlist") {Description = "File with list of WebAssembly import module names for PInvoke functions"};
```

Comment:
Could you manually check if these continue to work? Just to protect against changes in the meaning of the type argument...